GCTC-NTGC/gc-digital-talent
# Fix Total estimated candidates message in summary of filters

Resolves #3702.

### Steps to test
1. `cd frontend/talentsearch`
2. `npm run intl-compile`
3. `npm run production`
4. Navigate to /en/search/request
5. Verify text after _Request for pool candidates:_ is _no candidates_ for `null` candidateCount
6. Navigate to /fr/search/request
7. Verify text after _Demande de candidats pour le bassin :_ is _aucun candidat_ for `null` candidateCount
8. `npm run storybook`
9. Find **RequestForm** component
10. Test candidateCount values for 0, 1, 2 in English and French
